spec/jobs/sync_salesforce_contacts_job_spec.rb

```diff
@@ -17,8 +24,8 @@
 	end
 
 	it 'syncs one contact' do
-		SyncSalesforceContactsJob.new.perform('0034C00000T7QPQQA3')
+		SyncSalesforceContactsJob.new.perform('0037h00000YQofJAAT')
```

Comment:
at some point we need to look at stubbing these requests so we don't have to keep changing ids